about.html

```diff
@@ -11,61 +11,73 @@
 
 </head>
 
-<body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
 
+ 
+<body>
+  <div class="container">
+    <header>
+      
+      <nav>
```

Comment:
You did a good job creating the navbar, my only suggestion is to be mindful of indenting and spacing. Your code overall looks clean and easy to follow but in the <nav> element  you forgot to indent the tags inside of it